publons/version.xml

```diff
@@ -12,8 +12,8 @@
 <version>
 	<application>publons</application>
 	<type>plugins.generic</type>
-	<release>3.2.0.0</release>
-	<date>2019-8-06</date>
+	<release>3.2.1.0</release>
```

Comment:
I believe it should be 3.1.2 to align with OJS versioning as I mentioned in the issue

Reply:
> I believe it should be 3.1.2 to align with OJS versioning as I mentioned in the issue

This version is for the plugin itself, We shouldn't be decrementing our versioning here, as users expect to upgrade their plugin versions.